scripts/microbe_stage/organelle.as

```diff
@@ -839,6 +839,10 @@ class PlacedOrganelle : SpeciesStoredOrganelleType{
         //sceneNode.entity.setColour(colour)
     }
 
+    void hideEntity() {
+    auto renderNode = world.GetComponent_RenderNode(organelleEntity);
+    if(renderNode.Node !is null) renderNode.Node.removeFromParent();
```

Comment:
This should be (to have proper error checking, and formatted better, the scripts should follow what clang-format does for C++ code):
```
if(renderNode !is null && renderNode.Node !is null)
    renderNode.Node.removeFromParent();
```


Also the intendation is also a bit fishy on `organelle.hideEntity()`.